VOM_Fortran/VOM-docu/source/vom_namelist.rst

```diff
@@ -27,7 +27,7 @@ The file "vom_namelist" is an input file for the VOM, containing all settings fo
 +-----------------+----------------------------------------------------------------------------------------------------+-----------+
 |i_mdtf           | total dry mass of living tissues of trees per unit pc                                              | g/m^2     |
 +-----------------+----------------------------------------------------------------------------------------------------+-----------+
-|i_mqxtf          | total water storage capacity in living tissues of trees per unit md                                | kg/m^2    |
+|i_mqxtf          | total water storage capacity in living tissues of trees per unit md                                | -         |
```

Comment:
Is it really dimensionless? Then it should be "Relative", not "total".

Reply:
Yes, I think so. Else the units do not match here:

```
!     * Set vegetation parameters

      q_md   = o_cait * i_mdtf + o_mdstore !dry matter (kg/m2) 
      q_mqx  = q_md * i_mqxtf              !dry matter (kg/m2) * water storage cap. living matter  (-) = (kg/m2)
```

Reply:
My bad, it says "total ... per unit md", so it is kg/kg.